### DISCUSSION TITLE
Rootless: Composefs storage driver

### DISCUSSION THREAD

**Discussion by j1nx** [composefs/composefs#369] [General]
I am reading through all available information online to learn about composefs. It looks like with the upcoming kernel 6.12 (LTS?) things are looking great for erofs and/or composefs.

based on;
https://github.com/containers/storage/issues/2099

Am I correct in understanding that composefs is not yet available for rootless containers?

**Comment by cgwalters** [comment]
That's right, I think we can track things there if you're looking at podman specifically. If you're interested in unprivileged composefs for other use cases in theory we could have a tracker here too. Tentatively closing on that basis.
